diff --git a/dist/rollup-plugin-dts.mjs b/dist/rollup-plugin-dts.mjs
index 4a9412285c48c37d03340a086c771f8e61fd82ac..c73cba3bf47550f69011366e37d2ae974f0c9fc0 100644
--- a/dist/rollup-plugin-dts.mjs
+++ b/dist/rollup-plugin-dts.mjs
@@ -675,6 +675,8 @@ function preProcess({ sourceFile }) {
             const nextToken = children[idx + 1];
             const isPunctuation = nextToken.kind >= ts.SyntaxKind.FirstPunctuation && nextToken.kind <= ts.SyntaxKind.LastPunctuation;
             if (isPunctuation) {
+                const addSpace = code.slice(token.getEnd(), nextToken.getStart()) != " ";
+                code.appendLeft(nextToken.getStart(), `${addSpace ? " " : ""}${defaultExport}`);
                 code.appendLeft(nextToken.getStart(), defaultExport);
             }
             else {
